modules/rhel-compute-updating.adoc

```diff
@@ -76,9 +76,9 @@ You must not enable firewalld later. If you do, you cannot access {product-title
 [source,terminal]
 ----
 NAME                        STATUS                        ROLES    AGE    VERSION
-mycluster-control-plane-0   Ready                         master   145m   v1.20.0
-mycluster-control-plane-1   Ready                         master   145m   v1.20.0
-mycluster-control-plane-2   Ready                         master   145m   v1.20.0
+mycluster-control-plane-0   Ready                         master   145m   v1.21.0
+mycluster-control-plane-1   Ready                         master   145m   v1.21.0
+mycluster-control-plane-2   Ready                         master   145m   v1.21.0
 mycluster-rhel7-0           NotReady,SchedulingDisabled   worker   98m    v1.14.6+97c81d00e
 mycluster-rhel7-1           Ready                         worker   98m    v1.14.6+97c81d00e
 mycluster-rhel7-2           Ready                         worker   98m    v1.14.6+97c81d00e
```

Comment:
All the nodes version also should be updated to `v1.21.0`.

Reply:
Good catch thanks - updated!